package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "messagebird",
-  "version": "2.2.0",
+  "version": "3.2.0",
```

Comment:
Since this is not a breaking change it should probably be a `minor` version to follow semver

Reply:
Maybe skip the version bump in this PR as mentioned by Raymond here: https://github.com/messagebird/messagebird-nodejs/pull/24#discussion_r249010545